### PR TITLE
[3.10] Deprecation notice for tabstate

### PR DESCRIPTION
**Pull request by brianteeman**
Simple PR to add a deprecattion notice for JHtmlBehavior::tabstate

cc @wilsonge

### CODE REVIEW

File: libraries/cms/html/behavior.php [hunk 1]
```diff
@@ -883,9 +883,13 @@ protected static function _getJSObject($array = array())
 	 * @return  void
 	 *
 	 * @since   3.2
+	 *
+	 * @Deprecated 4.0 Use the custom element joomla-tab.
```
Comment (wilsonge) [inline libraries/cms/html/behavior.php]: lowercase on deprecated. Can we please add "in Joomla 4 use the custom element joomla-tab" as for now the custom elements don't exist in J3

Comment (brianteeman) [reply libraries/cms/html/behavior.php]: Done

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks and thanks for fixing the bit I missed on my phone

**Comment by wilsonge** [comment]
Np!

**Pull request merged**